app/javascript/components/my_osbb/News/ShowNews.module.scss

```diff
@@ -0,0 +1,32 @@
+.img_div {
+  width: 100%; 
+  text-align: center;
+}
+.news_img {
+  max-width: 100%;;
+  max-height: 300px;
+}
+
+.news_main_div{
+  width: 100vw;
+  margin-left: -8px;
+}
+
+.news_form {
+  margin: 0 auto;
+  width: 55%;
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 20px;
+}
+
+.title_content {
```

Comment:
Maybe you should to rename this class. Something like ".news..." to avoid UI bugs in future